src/types/index.d.ts

```diff
@@ -91,6 +91,7 @@ export type NewsStoryType = PublishedEntity & {
   entityId: number
   entityPath: string
   administration: Administration
+  breadcrumbs: any
```

Comment:
I think we know what types these will be (maybe `BreadcrumbItem[]` ?), per the testing snapshot:

```
  "breadcrumbs": Array [
    Object {
      "options": Array [],
      "title": "Home",
      "uri": "https://cms-7m0jwgfd3t0pt3txlgzvixowibrfagsq.ci.cms.va.gov/",
    },
  ],
  ```

Reply:
missed that on my cleanup, fixed